addons/hr_attendance/models/hr_attendance.py

```diff
@@ -132,14 +131,14 @@ def _compute_display_name(self):
             if not attendance.check_out:
                 attendance.display_name = _(
                     "From %s",
-                    format_datetime(self.env, attendance.check_in, dt_format="HH:mm"),
+                    format_time(self.env, attendance.check_in, time_format=None, lang_code=self.env.lang),
```

Comment:
can you add test case for changing time format on language?

Reply:
Hello @kais-odoo 
I've added the test case :slightly_smiling_face: